### ISSUE TITLE
Grunt errors with new install 

### ISSUE THREAD

**Issue by k33** [multiple-states/bread-butter#67]
I get Errors and Warnings when installing our grunt dependencies. running the latest version of Grunt. See http://gruntjs.com/blog/2016-04-04-grunt-1.0.0-released#peer-dependencies.

**Comment by k33** [comment]
Also getting this warning: WARN package.json bread-butter@1.0.0 No license field.

**Comment by k33** [comment]
I have gone through each grunt packages we install and the issues are related to grunt-pixrem

I then found this: 

"It's likely because one of the grunt plugins has "grunt": "~0.4.5" or some other bad peer dependency definition in their package.json. We've advised all plugins to update to >=0.4.5 (or remove) their peer dependency definition to avoid this.

They plugin author may have already updated their plugin in which case rm -rf node_modules && npm cache clean && npm i might fix it.

If not, look into which grunt plugin has grunt listed as a peer dependency. If it's ~0.4.5 or similar, it's likely incorrect. I would open an issue on their tracker and ask them to remove the peerDependency definition to grunt." SRC: https://github.com/gruntjs/grunt/issues/1537#issuecomment-235324836
